[Rough Draft] Document Relay Resolvers (#4547)
facebook/relay: website/docs/api-reference/relay-resolvers/limitations.md

---
id: limitations
title: "Limitations"
slug: /api-reference/relay-resolvers/limitations/
description: Limitations of Relay Resolvers
---

> Relay Resolvers are do have some limitations. Here we will collect a list of known limitations and provide alternatives where possible.

Reviewer: ```suggestion
Here are some known limitations of RelayResolvers and their alternatives, if any.
```
